msg/ko_KR.euckr/utils.msg

```diff
@@ -291,7 +291,7 @@ backupdb: 
   -t, --thread-count=COUNT     ���� �����ϴ� ������ ����; �⺻��: �ڵ�\n\
       --no-compress            ��� ����� �������� �ʽ�ϴ�; �⺻��: ����\n\
       --sleep-msecs=N          �� 1M �б� ���� N �и��ʸ� ���ϴ�; �⺻��: 0\n\
-  -k, --separate-keys          TDE�� ���Ǵ� Key File (_keys)� ���� �������� �ʰ� ������ ���Ϸ� �и��մϴ�.\n
+  -k, --separate-keys          TDE�� ���Ǵ� Key File (_keys)� ���� �������� �ʰ� ������ ���Ϸ� �и��մϴ�. \n
```

Comment:
unrelated

Reply:
@wrlawodms 
Right. Actually , I transformed from msg/ko_KR.utf8/utils.msg to msg/ko_KR.euckr/utils.msg. 

Because I cannot write down broken characters like "����" in the msg/ko_KR.euckr/utils.msg. 

So, I think this code should be not changed and applyinfo usage should be changed in this case. Right?

Reply:
@wrlawodms 
Unrelated code is rolled back. I will register about previous code as a issue.